irods_browser_app/static/css/site_base_irods.css

```diff
@@ -0,0 +1,83 @@
+#save_file {
+	padding: 3px;
+	cursor: pointer;
+	opacity:1;
+}
+
+#save_file:hover {
+	opacity: 0.7;
+}
+
+#irodsContent .modal-lg {
+	width: 570px;
+}
+
+#irodsContent .form-signin {
+	max-width: 865px;
+	padding: 5px;
+}
+
+#irods_content_label {
+	color: #357ebd;
+}
+
+#irodsContent .modal-body {
+	height: 342px;
+}
+
+#irodsContent #irods_structure {
+	width: 517px;
+	height: 300px;
+	padding: 10px;
+	float:left;
+	box-shadow:1px 1px 3px 1px #000;
+}
+
+#irodsContent #irods_structure #viewing input {
+	padding-left: 10px;
+	padding-right: 10px;
+	border-radius: 10px;
+	margin-left: 5px;
+	width: 277px;
+}
+
+#irodsContent #irods_structure #file_struct {
+	margin-top: 10px;
+	height: 244px;
+	overflow: scroll;
```

Comment:
Fixed all spacing issue with this file, consistently using tabs for indentation and using spaces for readability separation between phrases.